### PR TITLE
Roodkapje

### PR DESCRIPTION
**Pull request by Ebit22**
# Roodkapje

## Ebit22

**Author:**

## About your game

**What is your game about?**
Find the way to the house.

**How do you play your game?**
WASD to move, don't touch the wolfs.

## Code
Check off the items that are true. 
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [ ] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ ] The name of the image matches the name of your file. 

> Thanks for your PR!

### CODE REVIEW

**Review by grymmy — CHANGES_REQUESTED** [review]
Neat game!

Please change the file path of your game to the games/ directory per the submission instructions.

**Comment by grymmy** [comment]
I would also note that I found this game somewhat hard to play given the input keybindings (wsad didn't work as I'd expect)

**Comment by Ebit22** [comment]
I can't upload the file in the [/games directory]

File: Roodkapje.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+setMap(levels[level])
+
+setPushables({
+})
+
+onInput("s", () => {
+  getFirst(player).y += 1
+})
+onInput("a", () => {
```
Comment (Ebit22) [inline Roodkapje.js:297]:
```suggestion
onInput("w", () => {
```

File: Roodkapje.js [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+  getFirst(player).y -= 1
+})
+onInput("d", () => {
+  getFirst(player).x += 1
+})
+onInput("w", () => {
```
Comment (Ebit22) [inline Roodkapje.js:303]:
```suggestion
onInput("a", () => {
```

**Comment by grymmy** [comment]
Neat game!  Congratulations, if you're a teen, you can order a Sprig by filling out this form here: https://sprig-order.hackclub.dev/

**Review by grymmy — APPROVED** [review]

**Pull request merged**